pkg/reconciler/v1alpha1/taskrun/resources/input_resources.go

```diff
@@ -71,9 +71,6 @@ func AddInputResource(
 				if err != nil {
 					return nil, fmt.Errorf("task %q invalid Pipeline Resource: %q", task.Name, boundResource.ResourceRef.Name)
 				}
```

Comment:
we need to provide to the git clone command the revision or it will always get master I believe

Reply:
i think it's okay b/c the `revision` field is used

https://github.com/knative/build-pipeline/blob/0c9529db60cc053f9bcd14fc89a8e1454b873746/pkg/reconciler/v1alpha1/taskrun/resources/input_resources.go#L74

I'll make sure tho that the tests have a least one case with a custom revision!

Reply:
looks like there isn't one yet, i'll add it!

Reply:
yeah, that works.